tools/templates/actions/end.php

```diff
@@ -49,8 +49,12 @@
             case 'buttondropdown':
                 echo "\n</div> <!-- end of buttondropdown -->\n";
                 break;
+            case 'visibility':
+                echo "\n</div>\n";
```

Comment:
tu pourrais rajouter <!-- end of visibility --> avant le \n ?

Reply:
je l'ai pas mis exprès pour que quand on regarde le dom ça ne donne pas d'indice que quelque chose a été caché

Reply:
Ok, ça fait sens. On pourra le mettre quand on gérera la visibilité côté serveur.

Reply:
merci de l'avoir caché, mais faut juste espérer que personne ne désactive javascript alors :D